src/day-1-toy/fileio.py

```diff
@@ -1,12 +1,16 @@
 # Use open to open file "foo.txt" for reading
-
+myfile = open("./foo.txt")
 # Print all the lines in the file
```

Comment:
this works. here's a shorter way using read()
with open('foo.txt') as textFile:
text = textFile.read()
print(text)